### ISSUE TITLE
Hide "+" button from 3rd  level

### ISSUE THREAD

**Issue by archanatiwari** [Zicrael/ngx-tree-dnd#45]
Thanks for the wonderful template. 
I just want to know how to disable only the "+" button from third level ?

**Comment by Zicrael** [comment]
@archanatiwari 
you can use  "showExpandButton: false" in tree item options: 
``` 
options: {
    showExpandButton: false
},
```
Or you mean 'add new item" button?

**Comment by archanatiwari** [comment]
In your example you have created root->apple->iPhone. My requirement is to remove the '+' icon from the iPhone row.

**Comment by archanatiwari** [comment]
Like you have provided the options: {  showDeleteButton: false}
can I use  {  showAddButtons: false }

@Zicrael  Thank you

**Comment by Zicrael** [comment]
@archanatiwari  Yes sure. Sorry for late answer, I do it only when have free time.
